dashboard/app/controllers/test_status_controller.rb

```diff
@@ -0,0 +1,33 @@
+require 'aws-sdk'
+require 'date'
+
+# API that serves UI test status based on uploaded test run S3 logs and their
+# metadata.  See runner.rb, test_status.haml and test_status.js for more
+# information.
+class TestStatusController < ApplicationController
```

Comment:
nm I misread. I thought this was a REST API serving JSON only, but it looks like a regular controller backed by a view. Disregard.